db/migrate/20231104023500_create_likes.rb

```diff
@@ -0,0 +1,11 @@
+class CreateLikes < ActiveRecord::Migration[7.1]
+  def change
+    create_table :likes do |t|
+      t.integer :author_id
+      t.integer :post_id
```

Comment:
- [ ] Great effort working on the likes table. But the two columns that we should have in the table should be referenced from `post` and `user` and not be passed in as an integer. Below is an example of what is expected.
E.g
```ruby
  create_table :likes do |t|
    t.references :user, foreign_key: true 
    t.references :post, foreign_key: true
    t.timestamps
  end